ViewPumpSample/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 24
+    buildToolsVersion "24.0.0"
+
+    defaultConfig {
+        minSdkVersion 8
```

Comment:
I think we discussed changing this to API 14+ so we can remove the preHC LayoutInflater stuff.